gradlew

```diff
@@ -44,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
+DEFAULT_JVM_OPTS='"-Xmx2048m" "-Xms1024m"'
```

Comment:
Another 2GiB of RAM here means that we are using 6GiB of ram for the build system, if we account for a single daemon.

This seems completely ridiculous.  This is many, many times more RAM than the entire source code + build products (total of ~170MiB).